great_expectations/expectations/metrics/query_metrics/query_column.py

```diff
@@ -0,0 +1,57 @@
+from typing import Any, Dict, Optional, Union
+
+from great_expectations.execution_engine import (
+    ExecutionEngine,
+    PandasExecutionEngine,
+    SparkDFExecutionEngine,
+    SqlAlchemyExecutionEngine,
+)
+from great_expectations.execution_engine.execution_engine import MetricDomainTypes
+from great_expectations.expectations.metrics.import_manager import sa
+from great_expectations.expectations.metrics.metric_provider import metric_value
+from great_expectations.expectations.metrics.query_metric_provider import (
+    QueryMetricProvider,
+)
+
+
+class QueryColumn(QueryMetricProvider):
+    metric_name = "query.column"
+    value_keys = (
+        "column",
+        "query",
+    )
+
+    @metric_value(engine=SqlAlchemyExecutionEngine)
```

Comment:
I wonder if we might also want to think at some point about the Spark-SQL way to support Spark.

Reply:
🙌 Absolutely. I believe the pattern will be similar.